src/index.html

```diff
@@ -11,6 +11,16 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="search search--big" action="#"
+          data-qa="big">
```

Comment:
Need to fix code style everywhere
```suggestion
    <form
      class="search search--big"
      action="#"
      data-qa="big"
    >
```

Reply:
Does this really improves readability? For me personally it makes code longer, hence it's harder to traverse the code.